packages/compass-crud/src/components/table-view/cell-editor.jsx

```diff
@@ -236,15 +239,6 @@ class CellEditor extends React.Component {
     this.props.api.refreshCells({rowNodes: [this.props.node], force: true});
   }
 
-  focus() {
-    setTimeout(() => {
-      const container = ReactDOM.findDOMNode(this.props.reactContainer);
```

Comment:
Looks like this was unused, also it was using the deprecated `findDOMNode` so I removed.

```
Do not use findDOMNode. It doesn’t work with function components and is deprecated in StrictMode. See https://reactjs.org/docs/react-dom.html#finddomnode  react/no-find-dom-node
```

Reply:
You mean, the `.focus()` call didn’t work?

Reply:
Ah sorry, totally misworded that, yes the `.focus()` call didn't work. This function was called/used, but it wasn't doing anything.
I'll spend a bit and see if we can get this behavior as intended.

Reply:
Probably better we're not autofocusing here for now. I think we'll revisit the table separately and we can see if it's something we want then. As we're not losing functionality here compared to what was happening I think it's okay to remove. I can spend more time re-implementing this part if we're thinking it's useful.